back-end/helpers/loans-mock-data.js

```diff
@@ -0,0 +1,25 @@
+const mockData = {
```

Comment:
@puteulanus the file is already contains the mock name in it so, the variable name doesn't need to repeat it,
also if you don't destructure when importing it you can use it like:
```javascript
import mock from 'loans-mock-data'
...
res.send(mock.data)
```